### ISSUE TITLE
from_julian_date() throws SpatialException for dates >= "2022-12-31 00:00"

### ISSUE THREAD

**Issue by NiklasPhabian** [SpatioTemporal/pystare#120]
```python
import astropy.time, pystare, datetime

dt = datetime.datetime(2023,1,3)
dt_astro = astropy.time.Time(dt, scale='utc', format='datetime')

# This works fine:
pystare.tiv_from_datetime(dt) 

# This fails:
pystare.from_julian_date(dt_astro.jd1, dt_astro.jd2, scale='utc', forward_res=48, reverse_res=48)
```

```bash
terminate called after throwing an instance of 'SpatialException'
  what():  In TemporalIndex::fromJulianUTC, eraUtctai(...) failure.
Aborted (core dumped)
```

**Comment by NiklasPhabian** [comment]
All is fine up to 2022-12-30 23:59

**Comment by NiklasPhabian** [comment]
- dt = 2022-12-31 00:00 
- JD = 2459944.5 
- JD1= 2459944; JD2=0.5

**Comment by NiklasPhabian** [comment]
a little more isolated:

```python 
import astropy.time, pystare, datetime, numpy

dt = datetime.datetime(2023,1,3)
dt_astro = astropy.time.Time(dt, scale='utc', format='datetime')

pystare._core._from_JulianUTC(numpy.array([dt_astro.jd1]), numpy.array([dt_astro.jd2]), 48, 48)
```

**Comment by michaelleerilee** [comment]
This is corrected in [STARE Release 1.2.7](https://github.com/SpatioTemporal/STARE/releases/tag/1.2.7).

For pystare: update STARE and the existing pystare should work. Though in situations where the installation is statically linked, a re-linking may need to occur.
